docs/en/ingest-management/processors/processor-add_network_direction.asciidoc

```diff
@@ -0,0 +1,53 @@
+[[add_network_direction-processor]]
+= Add network direction
+
+++++
+<titleabbrev>add_network_direction</titleabbrev>
+++++
+
+The `add_network_direction` processor attempts to compute the perimeter-based
+network direction when given a source and destination IP address and a list of
+internal networks.
+
+[discrete]
+== Example
+
+[source,yaml]
+-------
+  - add_network_direction:
+      source: source.ip
+      destination: destination.ip
+      target: network.direction
+      internal_networks: [ private ]
+-------
+
+[discrete]
+== Configuration settings
+
+[options="header"]
+|===
+| Name | Required | Default | Description
+
+| `source`
+| Yes
+|
+| Source IP.
+
+| `destination`
+| Yes
+|
+| Destination IP.
+
+| `target`
+| Yes
+|
+| Target field where the network direction will be written. 
+
+| `internal_networks`
+| Yes
+|
+| List of internal networks. The value can contain either CIDR blocks or a list of special values enumerated in the network section of <<conditions>>.
+
+|===
+
+//REVIEWERS: Are all these fields required, or is there a default for `target`?
```

Comment:
Need input from reviewers on this question.

Reply:
Yes they're all required.